package-lock.json

```diff
@@ -0,0 +1,5171 @@
+{
+  "name": "react-native-photos-framework",
```

Comment:
Did you use `npm install` instead of `yarn install`?

Reply:
I used npm, out of habit! Will quick change to yarn